tests/_apis/team_fight_tactics/test_DataDragonApi.py

```diff
@@ -0,0 +1,194 @@
+from unittest.mock import MagicMock
+
+import pytest
+
+from riotwatcher._apis.team_fight_tactics import DataDragonApi
+
+@pytest.mark.lol
+@pytest.mark.unit
+class TestDataDragonApi:
+    def test_arenas(self):
+        mock_base_api = MagicMock()
+        expected_return = object()
+        mock_base_api.raw_request_static.return_value = expected_return
+
+        static_data = DataDragonApi(mock_base_api)
+
+        version = "234"
+        locale = "sdfasdf"
+
+        ret = static_data.arenas(version, locale)
+
+        mock_base_api.raw_request_static.assert_called_once_with(
+            f"https://ddragon.leagueoflegends.com/cdn/{version}/data/{locale}/tft-arena.json",
+            {},
+        )
```

Comment:
optional because legacy issues:

there is a more elegant way to not repeat this code in each function using a text fixture instead. See  [league matchv5 unit tests](https://github.com/pseudonym117/Riot-Watcher/blob/master/tests/_apis/league_of_legends/test_MatchApiV5.py) for an example.